modules/openxr/openxr_api.cpp

```diff
@@ -1411,6 +1415,11 @@ bool OpenXRAPI::poll_events() {
 				}
 				break;
 		}
+		for (OpenXRExtensionWrapper *wrapper : registered_extension_wrappers) {
+			if (wrapper->get_handled_events().has(runtimeEvent.type)) {
```

Comment:
We already have:
```
		for (OpenXRExtensionWrapper *wrapper : registered_extension_wrappers) {
			handled |= wrapper->on_event_polled(runtimeEvent);
		}
```
that handles events in wrappers. This is double.

Also whether we call into the wrapper with a `get_handled_events` function and then a `handle_event` function, or we just call a handler that returns false if we don't handle the event, and true if we had, I think the current solution is easier.